site/usage/applications.md

```diff
@@ -62,6 +62,8 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 
 ## Tools for Embedding Vega-Lite Visualizations
 
+
+- <span class="octicon octicon-star"></span>[Flourish](https://flourish.studio/2018/05/29/vega-lite-in-flourish/) - Visualization and Storytelling Platform
```

Comment:
Let's move it below embed and remove the star from Visdown since it has not received any updates in half a year.